diff -ur lms/lib/upgradedb/mysql.2010122000.php lms.new//lib/upgradedb/mysql.2010122000.php
--- lms/lib/upgradedb/mysql.2010122000.php	2011-01-17 11:38:43.000000000 +0100
+++ lms.new//lib/upgradedb/mysql.2010122000.php	2011-01-29 20:09:49.986391872 +0100
@@ -38,8 +38,7 @@
 $DB->Execute("ALTER TABLE cashimport ADD sourcefileid integer DEFAULT NULL
         REFERENCES sourcefiles (id) ON DELETE SET NULL ON UPDATE CASCADE");
 
-$DB->Execute("ALTER TABLE cashimport ALTER customerid DROP NOT NULL");
-$DB->Execute("ALTER TABLE cashimport ALTER customerid SET DEFAULT NULL");
+$DB->Execute("ALTER TABLE cashimport MODIFY customerid int(11) DEFAULT NULL");
 $DB->Execute("UPDATE cashimport SET customerid = NULL WHERE customerid NOT IN (SELECT id FROM customers)");
 $DB->Execute("ALTER TABLE cashimport ADD FOREIGN KEY (customerid)
         REFERENCES customers (id) ON DELETE SET NULL ON UPDATE CASCADE");
Tylko w lms.new//lib/upgradedb: mysql.2010122000.php~
