--- lms/daemon/modules/pinger/pinger.h.orig	2004-11-25 18:22:13.361727330 +0100
+++ lms/daemon/modules/pinger/pinger.h	2004-11-25 18:23:05.253060772 +0100
@@ -4,7 +4,7 @@
 #include <arpa/inet.h>
 #include <linux/if_arp.h>
 #include <features.h>    /* for the glibc version number */
-#if __GLIBC__ >= 2 && __GLIBC_MINOR__ > 3
+#if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 3
 #include <net/if.h>
 #include <netpacket/packet.h>
 #include <net/ethernet.h>     /* the L2 protocols */
