--- modules/promotionlist.php	2011/03/02 10:31:05	1.1
+++ modules/promotionlist.php	2011/03/21 07:57:57	1.2
@@ -21,14 +21,14 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
  *  USA.
  *
- *  $Id$
+ *  $Id$
  */
 
 $layout['pagetitle'] = trans('Promotions List');
 
 $promolist = $DB->GetAll('SELECT p.id, p.name, p.description, disabled,
 			(SELECT COUNT(*) FROM promotionschemas
-				WHERE p.id = promotionid) AS schemas,
+				WHERE p.id = promotionid) AS scs,
 			(SELECT COUNT(DISTINCT a.tariffid)
 			    FROM promotionassignments a
 				JOIN promotionschemas s ON (s.id = a.promotionschemaid)
