--- lms/index.php-u	2005-06-15 22:48:13.000000000 +0200
+++ lms/index.php	2005-06-15 22:53:52.000000000 +0200
@@ -82,14 +82,14 @@
 	$CONFIG[$key] = $val;
 
 // Check for configuration vars and set default values
-$CONFIG['directories']['sys_dir'] = (!isset($CONFIG['directories']['sys_dir']) ? getcwd() : $CONFIG['directories']['sys_dir']);
-$CONFIG['directories']['backup_dir'] = (!isset($CONFIG['directories']['backup_dir']) ? $CONFIG['directories']['sys_dir'].'/backups' : $CONFIG['directories']['backup_dir']);
+$CONFIG['directories']['sys_dir'] = (!isset($CONFIG['directories']['sys_dir']) ? '/usr/share/lms' : $CONFIG['directories']['sys_dir']);
+$CONFIG['directories']['backup_dir'] = (!isset($CONFIG['directories']['backup_dir']) ? '/var/lib/lms/backups' : $CONFIG['directories']['backup_dir']);
 $CONFIG['directories']['doc_dir'] = (!isset($CONFIG['directories']['doc_dir']) ? $CONFIG['directories']['sys_dir'].'/documents' : $CONFIG['directories']['doc_dir']);
 $CONFIG['directories']['lib_dir'] = (!isset($CONFIG['directories']['lib_dir']) ? $CONFIG['directories']['sys_dir'].'/lib' : $CONFIG['directories']['lib_dir']);
 $CONFIG['directories']['modules_dir'] = (!isset($CONFIG['directories']['modules_dir']) ? $CONFIG['directories']['sys_dir'].'/modules' : $CONFIG['directories']['modules_dir']);
 $CONFIG['directories']['config_templates_dir'] = (!isset($CONFIG['directories']['config_templates_dir']) ? $CONFIG['directories']['sys_dir'].'/config_templates' : $CONFIG['directories']['config_templates_dir']);
-$CONFIG['directories']['smarty_dir'] = (!isset($CONFIG['directories']['smarty_dir']) ? (is_readable('/usr/share/php/smarty/libs/Smarty.class.php') ? '/usr/share/php/smarty/libs' : $CONFIG['directories']['lib_dir'].'/Smarty') : $CONFIG['directories']['smarty_dir']);
-$CONFIG['directories']['smarty_compile_dir'] = (!isset($CONFIG['directories']['smarty_compile_dir']) ? $CONFIG['directories']['sys_dir'].'/templates_c' : $CONFIG['directories']['smarty_compile_dir']);
+$CONFIG['directories']['smarty_dir'] = (!isset($CONFIG['directories']['smarty_dir']) ? '/usr/share/php/Smarty' : $CONFIG['directories']['smarty_dir']);
+$CONFIG['directories']['smarty_compile_dir'] = (!isset($CONFIG['directories']['smarty_compile_dir']) ? '/var/lib/lms/templates_c/' : $CONFIG['directories']['smarty_compile_dir']);
 $CONFIG['directories']['smarty_templates_dir'] = (!isset($CONFIG['directories']['smarty_templates_dir']) ? $CONFIG['directories']['sys_dir'].'/templates' : $CONFIG['directories']['smarty_templates_dir']);
 
 foreach(lms_parse_ini_file($CONFIG['directories']['lib_dir'].'/config_defaults.ini', TRUE) as $section => $values)
--- lms/sample/lms.ini.orig	2005-05-13 13:36:19.000000000 +0200
+++ lms/sample/lms.ini	2005-06-15 23:04:30.000000000 +0200
@@ -268,7 +268,7 @@
 ; index.php, graphics, templates and the rest. By default index.php seek in filesystem
 ; using getcwd(), but better is to say him where hi is:
 
-; sys_dir          = /var/www/htdocs/lms/
+; sys_dir      = /usr/share/lms/www/
 
 ; Directory with "modules" of LMS. That is content of /modules directory.
 ; By default it is subdirectory modules in sys_dir
@@ -285,7 +285,7 @@
 ; If directory with backups will be accessible from WWW level, then anybody will
 ; can to access them without authorization.
 
-; backup_dir          = /var/backup/lms/
+; backup_dir          = /var/lib/lms/backups/
 
 ; In that directory LMS will store uploaded documents (files). 
 ; By default it is subdirectory documents in sys_dir.
@@ -299,7 +299,7 @@
 ; By default, subdirectory Smarty in lib_dir
 ; http://smarty.php.net
 
-; smarty_dir          = /usr/lib/php/Smarty
+; smarty_dir          = /usr/share/php/Smarty
 
 ; W pld jest domy¶lnie tutaj: (nie zapomnij o "poldek -i Smarty");
 ; In PLD-linux distribution it's here: (and don't forget about "poldek -i Smarty")
@@ -309,7 +309,7 @@
 ; Smarty's compilation directory. Place where Smarty compile templates.
 ; By default it is subdirectory templates_c in sys_dir.
 
-; smarty_compile_dir     = /var/smarty/compile/lms
+; smarty_compile_dir   = /var/lib/lms/templates_c
 
 ; Directory with Smarty's templates.
 ; By default it is subdirectory templates in sys_dir.
