--- lms/modules/traffic.php.orig	2004-11-01 13:19:21.403404075 +0100
+++ lms/modules/traffic.php	2004-11-01 13:21:32.709210849 +0100
@@ -27,28 +27,34 @@
 
 $bars = 1;
 
-if (isset($_GET['bar']) && isset($_POST['order']))
+if (isset($_GET['bar']) && isset($_POST['order'])) {
 	$_SESSION['trafficorder'] = $_POST['order'];
+	$_SESSION['trafficnet'] = $_POST['net'];
+}
 	
 switch($_GET['bar'])
 {
 	case 'hour':
-		$traffic = $LMS->Traffic( '?NOW?-(60*60)', '?NOW?', 0,
+		$traffic = $LMS->Traffic( '?NOW?-(60*60)', '?NOW?',
+			isset($_SESSION['trafficnet']) ?  $_SESSION['trafficnet'] : 0,
 			isset($_SESSION['trafficorder']) ? $_SESSION['trafficorder'] : 'download');
 	break;
 
 	case 'day':
-		$traffic = $LMS->Traffic( '?NOW?-(60*60*24)','?NOW?',  0,
+		$traffic = $LMS->Traffic( '?NOW?-(60*60*24)','?NOW?',
+			isset($_SESSION['trafficnet']) ?  $_SESSION['trafficnet'] : 0,
 			isset($_SESSION['trafficorder']) ? $_SESSION['trafficorder'] : 'download');
 	break;
 
 	case 'month':
-		$traffic = $LMS->Traffic( '?NOW?-(60*60*24*30)', '?NOW?', 0,
+		$traffic = $LMS->Traffic( '?NOW?-(60*60*24*30)', '?NOW?',
+			isset($_SESSION['trafficnet']) ?  $_SESSION['trafficnet'] : 0,
 			isset($_SESSION['trafficorder']) ? $_SESSION['trafficorder'] : 'download');
 	break;
 
 	case 'year':
-		$traffic = $LMS->Traffic( '?NOW?-(60*60*24*365)', '?NOW?', 0,
+		$traffic = $LMS->Traffic( '?NOW?-(60*60*24*365)', '?NOW?',
+			isset($_SESSION['trafficnet']) ?  $_SESSION['trafficnet'] : 0,
 			isset($_SESSION['trafficorder']) ? $_SESSION['trafficorder'] : 'download');
 	break;
 
