--- lms/index.php-u	2005-06-15 22:48:13.000000000 +0200
+++ lms/index.php	2005-06-15 22:53:52.000000000 +0200
@@ -85,13 +85,13 @@
 	$_CONFIG[$key] = $val;
 
 // Check for configuration vars and set default values
-$_CONFIG['directories']['sys_dir'] = (!isset($_CONFIG['directories']['sys_dir']) ? getcwd() : $_CONFIG['directories']['sys_dir']);
-$_CONFIG['directories']['backup_dir'] = (!isset($_CONFIG['directories']['backup_dir']) ? $_CONFIG['directories']['sys_dir'].'/backups' : $_CONFIG['directories']['backup_dir']);
+$_CONFIG['directories']['sys_dir'] = (!isset($_CONFIG['directories']['sys_dir']) ? '/usr/share/lms' : $_CONFIG['directories']['sys_dir']);
+$_CONFIG['directories']['backup_dir'] = (!isset($_CONFIG['directories']['backup_dir']) ? '/var/lib/lms/backups' : $_CONFIG['directories']['backup_dir']);
 $_CONFIG['directories']['lib_dir'] = (!isset($_CONFIG['directories']['lib_dir']) ? $_CONFIG['directories']['sys_dir'].'/lib' : $_CONFIG['directories']['lib_dir']);
 $_CONFIG['directories']['modules_dir'] = (!isset($_CONFIG['directories']['modules_dir']) ? $_CONFIG['directories']['sys_dir'].'/modules' : $_CONFIG['directories']['modules_dir']);
 $_CONFIG['directories']['config_templates_dir'] = (!isset($_CONFIG['directories']['config_templates_dir']) ? $_CONFIG['directories']['sys_dir'].'/config_templates' : $_CONFIG['directories']['config_templates_dir']);
-$_CONFIG['directories']['smarty_dir'] = (!isset($_CONFIG['directories']['smarty_dir']) ? (is_readable('/usr/share/php/smarty/libs/Smarty.class.php') ? '/usr/share/php/smarty/libs' : $_CONFIG['directories']['lib_dir'].'/Smarty') : $_CONFIG['directories']['smarty_dir']);
-$_CONFIG['directories']['smarty_compile_dir'] = (!isset($_CONFIG['directories']['smarty_compile_dir']) ? $_CONFIG['directories']['sys_dir'].'/templates_c' : $_CONFIG['directories']['smarty_compile_dir']);
+$_CONFIG['directories']['smarty_dir'] = (!isset($_CONFIG['directories']['smarty_dir']) ? '/usr/share/php/Smarty' : $_CONFIG['directories']['smarty_dir']);
+$_CONFIG['directories']['smarty_compile_dir'] = (!isset($_CONFIG['directories']['smarty_compile_dir']) ? '/var/lib/lms/templates_c/' : $_CONFIG['directories']['smarty_compile_dir']);
 $_CONFIG['directories']['smarty_templates_dir'] = (!isset($_CONFIG['directories']['smarty_templates_dir']) ? $_CONFIG['directories']['sys_dir'].'/templates' : $_CONFIG['directories']['smarty_templates_dir']);
 
 foreach(lms_parse_ini_file($_CONFIG['directories']['lib_dir'].'/config_defaults.ini', TRUE) as $section => $values)

--- lms/sample/lms.ini.orig	2005-05-13 13:36:19.000000000 +0200
+++ lms/sample/lms.ini	2005-06-15 23:04:30.000000000 +0200
@@ -253,7 +253,7 @@
 ; index.php, graphics, templates and the rest. By default index.php seek in filesystem
 ; using getcwd(), but better is to say him where hi is:
 
-; sys_dir          = /var/www/htdocs/lms/
+; sys_dir      = /usr/share/lms/www/
 
 ; Directory with "modules" of LMS. That is content of /modules directory.
 ; By default it is subdirectory modules in sys_dir
@@ -270,13 +270,14 @@
 ; If directory with backups will be accessible from WWW level, then anybody will
 ; can to access them without authorization.
 
-; backup_dir          = /var/backup/lms/
+; backup_dir       = /var/lib/lms/backups/
 
 ; Directory with Smarty library
 ; By default, subdirectory Smarty in lib_dir
 ; http://smarty.php.net
 
-; smarty_dir          = /usr/lib/php/Smarty
+; smarty_dir       = /usr/share/php/Smarty
+
 
 ; W pld jest domy¶lnie tutaj: (nie zapomnij o "poldek -i Smarty");
 ; smarty_dir      = /usr/share/pear/Smarty
@@ -284,7 +285,7 @@
 ; Smarty's compilation directory. Place where Smarty compile templates.
 ; By default it is subdirectory templates_c in sys_dir.
 
-; smarty_compile_dir     = /var/smarty/compile/lms
+; smarty_compile_dir   = /var/lib/lms/templates_c
 
 ; Directory with Smarty's templates.
 ; By default it is subdirectory templates in sys_dir.
