diff -urpN lms.orig/daemon/configure lms/daemon/configure
--- lms.orig/daemon/configure	2005-06-20 08:36:59.000000000 +0200
+++ lms/daemon/configure	2005-07-19 00:22:17.000000000 +0200
@@ -3,7 +3,7 @@
 # Initialize default variables
 prefix="/usr/local"
 incdir="/usr/include"
-libdir="/usr/lib"
+libdir="/usr/lib64"
 with_mysql=yes
 
 # Create config.status file
@@ -89,7 +89,7 @@ Options: [defaults in brackets after des
   --prefix=PREFIX	sets program and modules install location
 			(/usr/local), modules goes to PREFIX/lms/lib and main
 			program to PREFIX/lms/bin
-  --libdir=DIR		sets location of database library (/usr/lib)
+  --libdir=DIR		sets location of database library (/usr/lib64)
   --incdir=DIR		sets location of database include files (/usr/include)
   --inifile=FILE        configuration file - disables online configuration
 EOF
diff -urpN lms.orig/daemon/modules/ggnotify/Makefile.in lms/daemon/modules/ggnotify/Makefile.in
--- lms.orig/daemon/modules/ggnotify/Makefile.in	2005-05-01 09:31:23.000000000 +0200
+++ lms/daemon/modules/ggnotify/Makefile.in	2005-07-19 00:23:16.000000000 +0200
@@ -1,5 +1,5 @@
 # dla libgadu i ekg --------------
-LDFLAGS += -L/usr/lib
+LDFLAGS += -L/usr/lib64
 CFLAGS += -I/usr/src/ekg/lib
 # --------------------------------
 
@@ -18,4 +18,4 @@ clean:
 	rm -rf $(OUT)
 
 install:
-	install -c -m 755 $(OUT) $(MODULESDIR)
\ No newline at end of file
+	install -c -m 755 $(OUT) $(MODULESDIR)
