--- lms-orig/index.php	2011-03-18 09:59:23.000000000 +0100
+++ lms/index.php	2011-03-20 15:11:25.781918286 +0100
@@ -47,13 +47,13 @@
 $CONFIG = (array) parse_ini_file($CONFIG_FILE, true);
 
 // Check for configuration vars and set default values
-$CONFIG['directories']['sys_dir'] = (!isset($CONFIG['directories']['sys_dir']) ? getcwd() : $CONFIG['directories']['sys_dir']);
+$CONFIG['directories']['sys_dir'] = (!isset($CONFIG['directories']['sys_dir']) ? '/usr/share/lms' : $CONFIG['directories']['sys_dir']);
 $CONFIG['directories']['lib_dir'] = (!isset($CONFIG['directories']['lib_dir']) ? $CONFIG['directories']['sys_dir'].'/lib' : $CONFIG['directories']['lib_dir']);
-$CONFIG['directories']['doc_dir'] = (!isset($CONFIG['directories']['doc_dir']) ? $CONFIG['directories']['sys_dir'].'/documents' : $CONFIG['directories']['doc_dir']);
+$CONFIG['directories']['doc_dir'] = (!isset($CONFIG['directories']['doc_dir']) ? '/var/lib/lms/documents': $CONFIG['directories']['doc_dir']);
 $CONFIG['directories']['modules_dir'] = (!isset($CONFIG['directories']['modules_dir']) ? $CONFIG['directories']['sys_dir'].'/modules' : $CONFIG['directories']['modules_dir']);
-$CONFIG['directories']['backup_dir'] = (!isset($CONFIG['directories']['backup_dir']) ? $CONFIG['directories']['sys_dir'].'/backups' : $CONFIG['directories']['backup_dir']);
+$CONFIG['directories']['backup_dir'] = (!isset($CONFIG['directories']['backup_dir']) ? '/var/lib/lms/backups' : $CONFIG['directories']['backup_dir']);
 $CONFIG['directories']['config_templates_dir'] = (!isset($CONFIG['directories']['config_templates_dir']) ? $CONFIG['directories']['sys_dir'].'/config_templates' : $CONFIG['directories']['config_templates_dir']);
-$CONFIG['directories']['smarty_compile_dir'] = (!isset($CONFIG['directories']['smarty_compile_dir']) ? $CONFIG['directories']['sys_dir'].'/templates_c' : $CONFIG['directories']['smarty_compile_dir']);
+$CONFIG['directories']['smarty_compile_dir'] = (!isset($CONFIG['directories']['smarty_compile_dir']) ? '/var/lib/lms/templates_c' : $CONFIG['directories']['smarty_compile_dir']);
 $CONFIG['directories']['smarty_templates_dir'] = (!isset($CONFIG['directories']['smarty_templates_dir']) ? $CONFIG['directories']['sys_dir'].'/templates' : $CONFIG['directories']['smarty_templates_dir']);
 
 define('SYS_DIR', $CONFIG['directories']['sys_dir']);
--- lms-orig/sample/lms.ini	2009-04-08 13:59:42.000000000 +0200
+++ lms/sample/lms.ini	2011-03-20 15:11:25.785251771 +0100
@@ -228,7 +228,7 @@
 ; index.php, graphics, templates and the rest. By default index.php seek in filesystem
 ; using getcwd(), but better is to say him where hi is:
 
-; sys_dir          = /var/www/htdocs/lms/
+; sys_dir          = /usr/share/lms/www/
 
 ; Directory with "modules" of LMS. That is content of /modules directory.
 ; By default it is subdirectory modules in sys_dir
@@ -245,7 +245,7 @@
 ; If directory with backups will be accessible from WWW level, then anybody will
 ; can to access them without authorization.
 
-; backup_dir          = /var/backup/lms/
+; backup_dir          = /var/lib/lms/backups/
 
 ; In that directory LMS will store uploaded documents (files). 
 ; By default it is subdirectory documents in sys_dir.
@@ -253,17 +253,17 @@
 ; can to access them without authorization. Those files are renamed to their
 ; md5 sums.
 
-; doc_dir          = /usr/share/lms/documents/
+; doc_dir          = /var/lib/lms/documents/
 
 ; Smarty's compilation directory. Place where Smarty compile templates.
 ; By default it is subdirectory templates_c in sys_dir.
 
-; smarty_compile_dir     = /var/smarty/compile/lms
+; smarty_compile_dir     = /var/lib/lms/templates_c/
 
 ; Directory with Smarty's templates.
 ; By default it is subdirectory templates in sys_dir.
 
-; smarty_templates_dir     = /usr/share/lms/templates
+; smarty_templates_dir     = /usr/share/lms/templates/
 
 [mail]
 
--- lms-orig/contrib/customer/index.php	2011-03-18 09:59:23.000000000 +0100
+++ lms/contrib/customer/index.php	2011-03-20 15:12:00.340164058 +0100
@@ -36,11 +36,11 @@
 $CONFIG = (array) parse_ini_file($CONFIG_FILE, true);
 
 // Check for configuration vars and set default values
-$CONFIG['directories']['sys_dir'] = (! $CONFIG['directories']['sys_dir'] ? getcwd() : $CONFIG['directories']['sys_dir']);
-$CONFIG['directories']['backup_dir'] = (! $CONFIG['directories']['backup_dir'] ? $CONFIG['directories']['sys_dir'].'/backups' : $CONFIG['directories']['backup_dir']);
+$CONFIG['directories']['sys_dir'] = (! $CONFIG['directories']['sys_dir'] ? '/usr/share/lms' : $CONFIG['directories']['sys_dir']);
+$CONFIG['directories']['backup_dir'] = (! $CONFIG['directories']['backup_dir'] ? '/var/lib/lms/backups' : $CONFIG['directories']['backup_dir']);
 $CONFIG['directories']['lib_dir'] = (! $CONFIG['directories']['lib_dir'] ? $CONFIG['directories']['sys_dir'].'/lib' : $CONFIG['directories']['lib_dir']);
 $CONFIG['directories']['modules_dir'] = (! $CONFIG['directories']['modules_dir'] ? $CONFIG['directories']['sys_dir'].'/modules' : $CONFIG['directories']['modules_dir']);
-$CONFIG['directories']['smarty_compile_dir'] = (! $CONFIG['directories']['smarty_compile_dir'] ? $CONFIG['directories']['sys_dir'].'/templates_c' : $CONFIG['directories']['smarty_compile_dir']);
+$CONFIG['directories']['smarty_compile_dir'] = (! $CONFIG['directories']['smarty_compile_dir'] ? '/var/lib/lms/templates_c' : $CONFIG['directories']['smarty_compile_dir']);
 $CONFIG['directories']['smarty_templates_dir'] = (! $CONFIG['directories']['smarty_templates_dir'] ? $CONFIG['directories']['sys_dir'].'/templates' : $CONFIG['directories']['smarty_templates_dir']);
 
 define('SYS_DIR', $CONFIG['directories']['sys_dir']);
@@ -74,7 +74,7 @@
 
 // Initialize templates engine
 
-require_once(LIB_DIR.'/Smarty/Smarty.class.php');
+require_once(LIB_DIR.'/Smarty/Smarty.class.php');
 
 $SMARTY = new Smarty;
 
@@ -98,7 +98,7 @@
 $SMARTY->assign_by_ref('LANGDEFS', $LANGDEFS);
 $SMARTY->assign_by_ref('_ui_language', $LMS->ui_lang);
 $SMARTY->assign_by_ref('_language', $LMS->lang);
-$SMARTY->template_dir = getcwd();
+$SMARTY->template_dir = SMARTY_TEMPLATES_DIR;
 $SMARTY->compile_dir = SMARTY_COMPILE_DIR;
 @include('locale/'.$LMS->ui_lang.'/strings.php');
 
--- lms-orig/userpanel/index.php	2011-03-18 09:59:24.000000000 +0100
+++ lms/userpanel/index.php	2011-03-20 15:11:25.788585257 +0100
@@ -45,8 +45,8 @@
 	$CONFIG['directories']['sys_dir'] = $CONFIG['directories']['sys_dir'];
 $CONFIG['directories']['lib_dir'] = (!isset($CONFIG['directories']['lib_dir']) ? $CONFIG['directories']['sys_dir'].'/lib' : $CONFIG['directories']['lib_dir']);
 $CONFIG['directories']['modules_dir'] = (!isset($CONFIG['directories']['modules_dir']) ? $CONFIG['directories']['sys_dir'].'/modules' : $CONFIG['directories']['modules_dir']);
-$CONFIG['directories']['userpanel_dir'] = (!isset($CONFIG['directories']['userpanel_dir']) ? getcwd() : $CONFIG['directories']['userpanel_dir']);
-$CONFIG['directories']['smarty_compile_dir'] = $CONFIG['directories']['userpanel_dir'].'/templates_c';
+$CONFIG['directories']['userpanel_dir'] = (!isset($CONFIG['directories']['userpanel_dir']) ? $CONFIG['directories']['sys_dir'].'/userpanel'  : $CONFIG['directories']['userpanel_dir']);
+$CONFIG['directories']['smarty_compile_dir'] = '/var/lib/lms/userpanel/templates_c';
 
 define('USERPANEL_DIR', $CONFIG['directories']['userpanel_dir']);
 define('USERPANEL_LIB_DIR', USERPANEL_DIR.'/lib/');
@@ -77,7 +77,7 @@
 
 // Initialize templates engine (must be before locale settings)
 
-require_once(LIB_DIR.'/Smarty/Smarty.class.php');
+require_once('/usr/share/php/Smarty/Smarty.class.php');
 
 $SMARTY = new Smarty;
 
