diff -ur lms/daemon/configure lms.new//daemon/configure
--- lms/daemon/configure	2009-07-14 19:04:12.000000000 +0200
+++ lms.new//daemon/configure	2010-09-08 12:27:03.122006557 +0200
@@ -3,7 +3,7 @@
 # Initialize default variables
 prefix="/usr/local"
 incdir="/usr/include"
-libdir="/usr/lib"
+libdir="/usr/lib64"
 lmsbindir=""
 lmslibdir=""
 with_mysql=yes
@@ -98,7 +98,7 @@
 			program to PREFIX/lms/bin
   --lmsbindir=DIR	sets location of target LMS binaries (PREFIX/lms/bin)
   --lmslibdir=DIR	sets location of target LMS modules (PREFIX/lms/lib)
-  --libdir=DIR		sets location of database library (/usr/lib)
+  --libdir=DIR		sets location of database library (/usr/lib64)
   --incdir=DIR		sets location of database include files (/usr/include)
   --inifile=FILE        configuration file - disables online configuration
 EOF
diff -ur lms/daemon/modules/ggnotify/Makefile.in lms.new//daemon/modules/ggnotify/Makefile.in
--- lms/daemon/modules/ggnotify/Makefile.in	2007-11-26 12:06:48.000000000 +0100
+++ lms.new//daemon/modules/ggnotify/Makefile.in	2010-09-08 12:25:05.081888479 +0200
@@ -1,5 +1,5 @@
 # dla libgadu i ekg --------------
-LDFLAGS += -L/usr/lib
+LDFLAGS += -L/usr/lib64
 CFLAGS += -I/usr/src/ekg/lib
 LIBS += -lgadu
 # --------------------------------
